### ISSUE TITLE
Is this okay that you validate on test set?

### ISSUE THREAD

**Issue by BorisLestsov** [gjy3035/GCC-SFCN#25]
https://github.com/gjy3035/GCC-SFCN/blob/a5921d2b1e4a6aaa3019568bc4e4fca9d386970e/loading_data.py#L33 - here validation set is created from test set of UCF_QNRF. Is this okay?

**Comment by gjy3035** [comment]
we will add the following description:
In this code, the validation is directly on the test set. Strictly speaking, it is evaluated on the val set (randomly selected from the training set, which is adopted in the paper). Here, for a comparable reproduction (namely fixed splitting sets), this code directly adopts the test set for validation, which causes that the results of this code are better than that of our paper.
Thanks a lot!
